### PR TITLE
Problem: incorrect variable name in wallet-connect example (fix #299)

### LINKED ISSUE #299: Problem: incorrect variable name in wallet-connect example
change to correct name

### PR DESCRIPTION
**Pull request by leejw51crypto**
Solution: use correct name

### CODE REVIEW

**Comment by leejw51crypto** [comment]
header file check is not clang-tidy error
it's compiler error.

File: .github/workflows/cpp-lint.yml [hunk 1]
```diff
@@ -9,6 +9,10 @@ on:
 jobs:
   cpp-linter:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        compiler: [clang++-14]
```
Comment (tomtau) [inline .github/workflows/cpp-lint.yml]: https://github.com/cronos-labs/play-cpp-sdk/pull/298/files#r1041692307

**Review by tomtau — COMMENTED** [review]
maybe better to resolve the linting issues in a separate PR first?

**Comment by elecming** [comment]
We are doing the same thing... it has a lot of conflicts, yes, I agree to work on linter issue on another PR.

**Comment by elecming** [comment]
I deleted cpp-linter.yml on https://github.com/cronos-labs/play-cpp-sdk/pull/292 to avoid linter issues (it is incorrect setup as I mentioned), please help review and merge before fixing the linter issue. So that we would not break the build because the linter issue.

**Review by elecming — DISMISSED** [review]
Please check https://github.com/cronos-labs/play-cpp-sdk/pull/292 and merge it first.

File: .github/workflows/cpp-lint.yml (hunk 1 above)
Comment (leejw51crypto) [reply .github/workflows/cpp-lint.yml]: i'll remove

**Review by tomtau — APPROVED** [review]

**Review by elecming — APPROVED** [review]

**Pull request merged**